python/opae.admin/opae/admin/fpga.py

```diff
@@ -375,11 +375,10 @@ def reboot(self):
             raise IOError('bad completion code: 0x{:8x}'.format(ccode))
 
 
-class fpga_base(class_node):
+class fpga_base(sysfs_device):
     FME_PATTERN = 'intel-fpga-fme.*'
```

Comment:
Remove Production driver sysfs patterns

Reply:
I'll do that in a later PR where I plan to refactor a lot of this.